### ISSUE TITLE
*** PLEASE MERGE PULL REQUEST 186 ***

### ISSUE THREAD

**Issue by frankhale** [frankhale/hbs#1]
IT IS ABSOLUTELY OBNOXIOUS THAT YOU'D [UPSTREAM] LIMIT COMMENTS AND PULL REQUESTS TO COLLABORATORS KNOWING FULL WELL THAT WE NEED PULL REQUEST 186 TO BE MERGED SO THAT NPM WILL STOP COMPLAINING ABOUT SECURITY ISSUES. WOW!

https://github.com/pillarjs/hbs/pull/186

**Comment by frankhale** [comment]
WHY HAS PULL REQUEST 186 SAT FOR 27 DAYS?

**Comment by frankhale** [comment]
Reference https://github.com/pillarjs/hbs/issues/185

**Comment by martinszy** [comment]
+1 please merge it

**Comment by frankhale** [comment]
Just a FYI, This is a troll fork because the upstream disabled comments, pull requests. This is here so that the original pull request has a link back to here asking the devs to merge the pull request.

**Comment by martinszy** [comment]
I know but I couldn't comment anywhere else to support the cause. The package has almost 80.000 weekly downloads, that's a lot of broken websites!

**Comment by frankhale** [comment]
Yeah it's seriously outrageous that the pull request that would fix this has been sitting for a month. All they gotta do is push the button to merge it. LOL!

**Comment by ilovespicynoodles** [comment]
Why has this still not been fixed? My whole website depends on hbs and it doesn't work because of the security issue.

**Comment by frankhale** [comment]
This is a troll fork created by me because the upstream turned off
contributions. Doing this was the only way to link an issue with the
upstream pull request. Soon as they merge this fork will be discarded.
